### PR TITLE
line length shortened, example call added

### PR DESCRIPTION
**Pull request by lobingera**
see discussion in https://github.com/JuliaLang/BinDeps.jl/pull/270

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,11 +135,25 @@ cairo = library_dependency("cairo", aliases = ["libcairo-2", "libcairo"], depend
     e.g. whether or not has the correct version. To do so, pass a function that takes 
     (name,handle) as an argument and returns `true` if the library is usable and `false` 
     it not. The `name` argument is either an absolute path or the library name if it is a
-    global system library, while the handle is a handle that may be passed to  `dlsym` to check library symbols or the return value of function. Note however that it 
-    is invalid to store the `handle`. Instead, use the `@load_dependencies` macro (see below).
+    global system library, while the handle is a handle that may be passed to `dlsym` to 
+    check library symbols or the return value of function. Note however that it
```
Comment (tkelman) [inline README.md]: return value of a function

File: README.md [hunk 2]
```diff
@@ -135,11 +135,25 @@ cairo = library_dependency("cairo", aliases = ["libcairo-2", "libcairo"], depend
     e.g. whether or not has the correct version. To do so, pass a function that takes 
     (name,handle) as an argument and returns `true` if the library is usable and `false` 
     it not. The `name` argument is either an absolute path or the library name if it is a
-    global system library, while the handle is a handle that may be passed to  `dlsym` to check library symbols or the return value of function. Note however that it 
-    is invalid to store the `handle`. Instead, use the `@load_dependencies` macro (see below).
+    global system library, while the handle is a handle that may be passed to `dlsym` to 
+    check library symbols or the return value of function. Note however that it 
+    is invalid to store the `handle`. Instead, use the `@load_dependencies` macro
```
Comment (tkelman) [inline README.md]: the only mentions of this have been removed, it's not recommended any more

Comment (lobingera) [inline README.md]: What has been removed?

Comment (tkelman) [inline README.md]: the documentation of load_dependencies

Comment (lobingera) [inline README.md]: Ahh

**Comment by musm** [comment]
Btw @lobingera hdf5 deps is a good example also

**Pull request merged**